apps/permissions/src/app/access-denied.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ButtonComponent } from './button.component';
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterLink } from '@angular/router';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'access-denied',
+  standalone: true,
+  imports: [CommonModule, ButtonComponent, RouterLink],
+  template: `
+    <p class="text-red-900 text-7xl">Access Denied!</p>
+    <button app-button [routerLink]="['/']">Back to login</button>
```

Comment:
when passing a string. you don't need to bind your prop with [], you can simply write:
`routerLink='/'` 

But that's just a preference, I find it easier to read